tests/conftest.py

```diff
@@ -2,34 +2,35 @@
 #
 # SPDX-License-Identifier: MIT
 
-import os
 import shutil
 
 from pathlib import Path
+from tempfile import TemporaryDirectory
 
 import pytest
 
 
-@pytest.fixture(name="project_directory_uri")
-def _project_directory_uri(tmp_path):
-    leading_slashes = "//" if os.sep == "/" else "///"
-    return f"file:{leading_slashes}{tmp_path.as_posix()}/plugin"
+@pytest.fixture(scope="session")
+def plugin_dir():
+    with TemporaryDirectory() as d:
+        directory = Path(d, "plugin")
+        shutil.copytree(Path.cwd() / "src", directory / "src")
+        for fn in ["pyproject.toml", "README.md", "CHANGELOG.md"]:
+            shutil.copy(Path.cwd() / fn, directory / fn)
 
+        yield directory.resolve()
 
-@pytest.fixture(name="new_project")
-def new_project(project_directory_uri, tmp_path, monkeypatch):
-    shutil.copytree(Path.cwd() / "src", tmp_path / "plugin" / "src")
-    for fn in ["pyproject.toml", "README.md", "CHANGELOG.md"]:
-        shutil.copy(Path.cwd() / fn, tmp_path / "plugin" / fn)
 
+@pytest.fixture(name="new_project")
+def new_project(plugin_dir, tmp_path, monkeypatch):
```

Comment:
Please prefix with _. The point is to avoid having a symbol of the same name as the fixture itself to prevent using it without having it as a parameter and getting a weird error message.